### DISCUSSION TITLE
Hope someone help me soon

### DISCUSSION THREAD

**Discussion by undyingkevin** [serenity-is/Serenity#7282] [Help]
My client want me to integrate React(azure-devops-ui) into serenity.is platform.
He has got startsharp premium and share it with me.
But the thing is that it is not working correctly. He gave me this extension. StartSharp.Template_v8.7.1.0.vsix
I already create the startsharp starter on visual studio using this extension. But this error happened.
How can I resolve this problem?
![error](https://github.com/user-attachments/assets/827b08aa-cd12-439d-8b3f-6f338025d432)

**Comment by edwardch** [comment]
Duplicate of [7279](https://github.com/orgs/serenity-is/discussions/7279)

For StartSharp-related issues you'll need to log into your StartSharp dashboard to get your instructions to load the pro packages. For issues you can post on the StartSharp repo. 

Ps. I am only a part of the community and it is encouraged to check the documentation and old discussions & issues first. Titles like "Urgent/help now" don't typically get prioritised in any online community so it is encouraged to write questions that are a bit more descriptive and show that you have put in the effort (in this case).

Hope that helps!
